cmd/cli/support_bugreport.go

```diff
@@ -103,6 +111,7 @@ func newSupportBugReportCmd(config *action.Configuration, stdout io.Writer, stde
 	f.StringSliceVar(&bugReportCmd.appNamespaces, "app-namespaces", nil, "Application namespaces")
 	f.StringSliceVar(&bugReportCmd.appDeployments, "app-deployments", nil, "Application deployments: <namespace>/<deployment>")
 	f.StringSliceVar(&bugReportCmd.appPods, "app-pods", nil, "Application pods: <namespace>/<pod>")
+	f.BoolVar(&bugReportCmd.collectIngress, "ingress", false, "Collect ingress")
```

Comment:
Just curious why this flag is required, and if required why not default ingress log collection to true. By default, I think it would be beneficial to collect as many logs as possible.

Reply:
Good question, it could go either way in my opinion. I figured it's best to have it separated out in case there is a user that doesn't want to include certain ingress-related data and logging. But I'm not against removing this flag and making it always collect ingress.